routes/queue.js

```diff
@@ -1,13 +1,14 @@
+const logger = require('winston');
 const messageBroker = require('../lib/messageBroker');
 
 module.exports = (app) => {
   app.get('/queue', (req, res) => {
     messageBroker.publish('Hello from Digital Paper Edit API', (err, data) => {
       if (err) {
-        console.log('Error from publish: ', err);
+        logger.error(`Error from queue.js: ${ err }`);
```

Comment:
queue.js maybe sounds like a route error rather than lib. Could tweak the error message later!

Reply:
Cool beans! Sounds good :) I think maybe they could all be refactored with JSON anyways to make them a little easier to parse?